buildenv/jenkins/jobs/builds/Build-linux_riscv64_cross

```diff
@@ -0,0 +1,74 @@
+def setBuildStatus(String message, String state, String sha) {
+    context = "continuous-integration/eclipse-omr/branch/linux_riscv64"
+    step([
+        $class: "GitHubCommitStatusSetter",
+        reposSource: [$class: "ManuallyEnteredRepositorySource", url: "https://github.com/eclipse/omr"],
+        contextSource: [$class: "ManuallyEnteredCommitContextSource", context: context],
+        errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
+        commitShaSource: [$class: "ManuallyEnteredShaSource", sha: sha ],
+        statusBackrefSource: [$class: "ManuallyEnteredBackrefSource", backref: "${BUILD_URL}flowGraphTable/"],
+        statusResultSource: [$class: "ConditionalStatusResultSource", results: [[$class: "AnyBuildResult", message: message, state: state]] ]
+    ]);
+}
+
+pipeline {
+    agent{label 'linux && x86 && compile:riscv64'}
+    environment {
+        PATH = "/usr/lib/ccache/:$PATH"
+        GTEST_COLOR = "0"
+    }
+    stages {
+        stage('Get Sources') {
+            steps {
+                timestamps {
+                    checkout scm
+                    setBuildStatus("In Progress","PENDING","${env.GIT_COMMIT}")
+                }
+            }
+        }
+        stage('Build') {
+            steps {
+                timestamps {
+                    echo 'Output CCACHE stats before running and clear them'
+                    sh '''ccache -s -z'''
+
+                    dir('build_native') {
+                        echo 'Configure...'
+                        sh '''cmake ..'''
```

Comment:
You should be able to get by with a smaller set of features, I think:
```
-DOMR_THREAD=OFF
-DOMR_PORT=OFF
-DOMR_OMRSIG=OFF
-DOMR_GC=OFF
-DOMR_FVTEST=OFF
-DOMR_EXAMPLE=OFF
```

Reply:
@rwy0717 Great, thanks! `-DOMR_EXAMPLE=` is actually needed, otherwise I get following error: 
```
CUDA_TOOLKIT_ROOT_DIR not found or specified
-- Could NOT find CUDA (missing: CUDA_TOOLKIT_ROOT_DIR CUDA_NVCC_EXECUTABLE CUDA_INCLUDE_DIRS CUDA_CUDART_LIBRARY) 
CMake Error at cmake/modules/OmrAssert.cmake:58 (message):
  Assertion failed: OMR_UTIL_GLUE_TARGET must be set.
Call Stack (most recent call first):
  util/omrutil/CMakeLists.txt:23 (omr_assert)


-- Configuring incomplete, errors occurred!
```